db/migrate/20221018191704_add_author_id_to_likes.rb

```diff
@@ -0,0 +1,6 @@
+class AddAuthorIdToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_column :likes, :author_id, :bigint
+    add_index :likes, :author_id
+  end
```

Comment:
- [ ] there should be a reference to author and post.

Reply:
Hey @mikethreels did you check the `Schema.rb` ?

Reply:
@raminka13, my apologies i think i made a mistake, please request another review